sphinx/source/docs/user_guide/examples/layout_vertical.py

```diff
@@ -1,3 +1,10 @@
+"""
+This is an example doc string
+
+* Foo
+* Bar
+
+"""
```

Comment:
Here's an example of a docstring that can be added to the top of an example. The RST content will *not* currently be renderered where the `bokeh-embed` happens, but *will* be rendered in the dedicated page that gets generated for this example. (Tho maybe that should be configurable in the directive). But this makes it super easy, e.g. to add indexing directives, so that all the examples can be indexed better.